### PR TITLE
feat: lock/unlock return success object

### PR DESCRIPTION
**Pull request by zespinosa97**

### CODE REVIEW

File: smartcar/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = '2.0.1'
+__version__ = '2.0.2'
```
Comment (KarthikBhaskara) [inline smartcar/__init__.py]: you will probably need to update docs!

Comment (gurpreetatwal) [inline smartcar/__init__.py]: this should be a breaking change as well

File: README.md [hunk 2]
```diff
@@ -293,10 +293,20 @@ wish to make requests to it
...
+#### Return
+| Type               | Description         |
+|:------------------ |:--------------------|
+| Dictionary.`status`| Set to success on successful request. |
+
 ### `lock(self)`
 
 Lock the vehicle.
 
+#### Return
```
Comment (KarthikBhaskara) [inline README.md]: can you add a `throws` argument so the reader knows what happens on an unsuccesful request

File: tests/test_vehicle.py [hunk 3]
```diff
@@ -146,10 +146,22 @@ def test_disconnect(self):
...
     def test_lock(self):
-        self.queue('POST', 'security')
-        self.check(self.vehicle.lock(), action='LOCK')
+        data = {
+            'status': 'success'
+        }
+        self.queue('POST', 'security', body=data)
+        
+        response = self.vehicle.lock()
+        self.check(response, action='LOCK')
```
Comment (KarthikBhaskara) [inline tests/test_vehicle.py]: what does `.check` do?

Comment (zespinosa97) [reply tests/test_vehicle.py]: it checks the authorization header

File: README.md [hunk 4]
```diff
@@ -297,6 +297,8 @@ Unlock the vehicle.
 | Type               | Description         |
 |:------------------ |:--------------------|
 | Dictionary.`status`| Set to success on successful request. |
+#### Throws
+<code>SmartcarError</code> on unsuccessful request
```
Comment (KarthikBhaskara) [inline README.md]: called `SmartcarException` in Python.

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html it's also `raise` in python. Take a look at this link for the standards - https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html.

you may want to add the `raise` above the code block too!

**Pull request merged**